src/opm/output/eclipse/AggregateGroupData.cpp

```diff
@@ -660,8 +661,10 @@ void injectionGroup(const Opm::Schedule&     sched,
                     const std::map<Opm::Group::InjectionCMode, int>& cmodeToNum,
                     IGrpArray&               iGrp)
 {
+    using IGroup = ::Opm::RestartIO::Helpers::VectorItems::IGroup::index;
     const bool is_field = group.name() == "FIELD";
     auto group_parent_list = groupParentSeqIndex(sched, group, simStep);
+    using IGroup = ::Opm::RestartIO::Helpers::VectorItems::IGroup::index;
```

Comment:
Do you need two of these or is this just a merge/rebase conflict resolution?

Reply:
Just a rebase - I'll fix

Reply:
Ehh - they are both needed; in different functions.

Reply:
> Ehh - they are both needed; in different functions.

I was thinking more about the fact that this function has [two](https://github.com/joakim-hove/opm-common/blob/f9a52791a4429f691f23b67d261a02ae93a2d2cf/src/opm/output/eclipse/AggregateGroupData.cpp#L664) [copies](https://github.com/joakim-hove/opm-common/blob/f9a52791a4429f691f23b67d261a02ae93a2d2cf/src/opm/output/eclipse/AggregateGroupData.cpp#L667) of the same `using IGroup = ...` statement.